playwright-e2e/dss/component/input.ts

```diff
@@ -37,8 +38,12 @@ export default class Input extends WidgetBase {
    * @param opts
    * @returns {Promise<void>}
    */
-  async fill(value: string | number, opts?: { dropdownOption: string, type?: boolean }): Promise<void> {
-    const useType = opts?.type ? opts.type : false;
+  async fill(value: string | number, opts: { dropdownOption?: string, type?: boolean, nth?: number, waitForSaveRequest?: boolean } = {}): Promise<void> {
```

Comment:
Add new parameter to include waiting of `/answers` request as some input fields have this request automatically triggered when out of focus.